ui/app/AppLayouts/Profile/views/wallet/PermissionsListView.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.14
```

Comment:
Should this not be moved in shared? is browser accessing wallet ok?

Reply:
@alaibe  yes, the folder structure suitable for it. Cos it's wallet part its logical :)